### DISCUSSION TITLE
Tool for Loadbalancing, Connection pooling, Automatic Failover

### DISCUSSION THREAD

**Discussion by Rammurthy5** [pgEdge/cli#18] [Q&A]
I am using PgEdge nodes setup in two different regions. Now exploring PgPool2 to fulfil Loadbalancing, Connection pooling, Automatic Failover. 
May i know if there's something from PgEdge that you offer to solve this, please?

**Comment by luss** [comment]
Hi RAM,

We stroingly recommend pgCat.  It's what we use for enterprise customers.  It's efficient (like pgBouncer) at pooling and it haas efficient sharding and routing capabilities.
